.github/workflows/ci.yml

```diff
@@ -98,7 +98,7 @@ jobs:
                 curl -L -O https://tiker.net/ci-support-v0
                 . ./ci-support-v0
                 build_py_project_in_conda_env
-                conda install matplotlib ipykernel nbconvert
+                pip install matplotlib ipykernel nbconvert
```

Comment:
```suggestion
                with_echo pip install matplotlib ipykernel nbconvert
```
?

Reply:
Done in https://github.com/inducer/loopy/compare/bdaa4bda4875aa709d1459f9c5fe4e4583dfce79..891401b902c1425efbf9373cb38e67182e56933d